universal-application-tool-0.0.1/app/views/dev/AzureViewStorageStrategy.java

```diff
@@ -0,0 +1,48 @@
+package views.dev;
```

Comment:
suggestion (blocking): move out of dev package. dev should only be for classes that are used in the development environment.

Reply:
I believe we discussed offline and this should stay as is.

Reply:
Yes, thanks!